watchtower/wtdb/client_db.go

```diff
@@ -68,6 +69,12 @@ var (
 	//    chan-id => start -> end
 	cSessionAckRangeIndex = []byte("client-session-ack-range-index")
 
+	// cSessionRogueUpdateCount is a sub-bucket of cSessionBkt storing the
```

Comment:
was wondering how this sub-bucket was cleaned, then realized this is a key rather than a sub-bucket. Think we should update the docs here.

Reply:
Missed this but been thinking if this were a sub-bucket we could just use a (kvdb provided) sequence too, although i like that it's just a key given there're no plans to add more related keys.

Reply:
oops! docs updated 👍